packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -131,10 +132,14 @@ export function OpenAI({
   return result;
 }
 
-const getEncoder = _.once(() => getEncoding('cl100k_base'));
+// Preload the tokenizer to avoid a large delay on first use.
+const tiktoken = new Tiktoken(cl100k_base);
+export const tokenizer = {
+  encode: (text: string) => tiktoken.encode(text),
+  decode: (tokens: number[]) => tiktoken.decode(tokens),
+};
```

Comment:
I'm confused, how is the `tiktoken` object different from the `tokenizer` object?
Are you just trying to make the interface cleaner by removing access to other functions or is this needed to trick TypeScript somehow?

Reply:
Relatedly (maybe answered by the other question): why is the first object called `tiktoken` and not something like `openAITikToken` or `cl100kTikToken`?

Reply:
1) Since we're now exporting this, I wanted to encapsulate (i.e. narrow) the Tiktoken interface.
2) Simply in the spirit of `const foo = new Foo(bar)`

Reply:
Okay works for me.